baby_bingo.py

```diff
@@ -0,0 +1,60 @@
+
+########## SUPPORTING PACKAGES ##########
+import random
+import re
+import sys
+
+
+########## SUPPORTING FUNCTIONS ##########
+
+def remove_number_prefix(my_string):
+    m = re.match('^[0-9]*[.]\s(.*)', my_string)
+    return(m.group(1))
+
+def remove_rating_suffix(my_string):
+    m = re.match('([a-zA-Z]*)\s(No change)*[+-]*[0-9]*', my_string)
+    return m.group(1)
+
+
+########## CODE TO BE EXECUTED ##########
+
+baby_names = [] # array that will store the baby names
+bingo_card_dim = 4 # the number of rows and columns the bingo card will include
+num_cards_needed = 15 # the number of bingo cards needed
+
+# open, read in, and close the file
+in_file = open('baby_names.txt', 'r')

```

Comment:
the file reading block could be moved into it's own function which returns the name list.